ADALiOS/ADALiOS/ADAuthenticationBroker.m

```diff
@@ -127,34 +127,18 @@ + (void)setResourcePath:(NSString *)resourcePath
 // cannot be loaded.
 + (NSBundle *)frameworkBundle
 {
-    static NSBundle       *bundle     = nil;
-    static dispatch_once_t predicate;
+    static dispatch_once_t once;
+    static NSBundle* bundle = nil;
     
-    @synchronized(self)
-    {
-        dispatch_once( &predicate,
-                      ^{
-
-                          NSString* mainBundlePath      = [[NSBundle mainBundle] resourcePath];
-                          AD_LOG_VERBOSE_F(@"Resources Loading", @"Attempting to load resources from: %@", mainBundlePath);
-                          NSString* frameworkBundlePath = nil;
-                          
-                          if ( _resourcePath != nil )
```

Comment:
Oh, never mind, [[NSBundle mainBundle] pathForResource:@"ADALiOS" ofType:@"bundle"]; is so powerful that it can always find the bundle now matter how :)